src/assets/json/communities.json

```
@@ -2151,5 +2151,27 @@
         "name": "Helen Obogeanu"
       }
     ]
+  },
+  "Angular Dominicana": {
+    "id": 88,
+    "name": "Angular Dominicana",
+    "position": {
+      "lng": -69.929611,
+      "lat": 18.483402
+    },
+    "city": "Santo Domingo",
+    "twitter": "https://twitter.com/ngdominican",
+    "web": "https://ngdominican.com",
+    "description": "Comunidad de desarrollo web y desarrolladores Angular de República Dominicana 🇩🇴",
```

are these characters deliberated?

```suggestion
    "description": "Comunidad de desarrollo web y desarrolladores Angular de República Dominicana",
```

---

Resolved